src/format.rs

```diff
@@ -313,6 +319,13 @@ impl PayloadParams {
             score = score.saturating_sub(2);
         }
 
+        // If neither value is specified both sides should assume DTX is not used as this is
+        // the default.
+        let either_dtx_specified = c0.format.use_dtx.is_some() || c1.format.use_dtx.is_some();
+        if either_dtx_specified && c0.format.use_dtx != c1.format.use_dtx {
+            score = score.saturating_sub(1);
+        }
```

Comment:
I wasn't sure how much to lower the score for a DTX mismatch. I notice `minptime` lowers it by 1 and `useinbandfec` lowers it by 2. Do they need to be distinct penalties?

Reply:
I think I might have aimed for making it deterministic what the sorting order is.

I.e. `minptime` and `use_dtx` shouldn't result in the same score.

Reply:
Hmm, if I had to put an ordering, then, I think DTX differences should be the biggest penalty, due to both the large difference in packet volume (and resulting bandwidth) between having it on vs. off, and the claimed small ding in audio quality.

If we want a deterministic sorting order, then, it sounds like we need to decrease the score by 4 in that case, since 3 would overlap with a simultaneous difference in `minptime` and `useinbandfec`. I'll make that change now.